website/thaliawebsite/static/css/base.scss

```diff
@@ -332,14 +332,15 @@ img {
         height: 100%;
 
         display: flex;
-        align-items: center;
+        justify-content: flex-start;
+        align-items: stretch;
+        align-content: stretch;
```

Comment:
How does this affect all other grid items with images?

Reply:
<img width="1173" alt="Screenshot 2022-01-23 at 12 35 03" src="https://user-images.githubusercontent.com/1799914/150676452-fba75f3a-919a-4d9e-83c4-218ed9820e22.png">

Quick copy paste in my inspector on the partners page and I say this breaks stuff.

Reply:
Too bad, I didn't think about the the partner page

Reply:
Thabloid and documents use a custom height grid-item too :)
Maybe take a look at that as well if you want to change this